test-loop-tests/src/utils/peer_manager_actor.rs

```diff
@@ -81,9 +94,33 @@ type NetworkRequestHandler = Box<dyn Fn(NetworkRequests) -> Option<NetworkReques
 /// - Override handler to modify data and simulate malicious behavior.
 pub struct TestLoopPeerManagerActor {
     handlers: Vec<NetworkRequestHandler>,
+
+    client_sender: ClientSenderForTestLoopNetwork,
+    genesis_id: GenesisId,
+    last_block_headers: HashMap<PeerInfo, BlockHeader>,
 }
 
-impl Actor for TestLoopPeerManagerActor {}
+impl Actor for TestLoopPeerManagerActor {
+    fn start_actor(&mut self, ctx: &mut dyn near_async::futures::DelayedActionRunner<Self>) {
+        const PUSH_NETWORK_INFO_PERIOD: Duration = Duration::milliseconds(100);
```

Comment:
Is 100 ms too low given block production time is ~1.2 sec? The downside of a small number would be that visualizer would be flooded with this message.

Reply:
I increased it to 300ms, with 500ms epoch_sync tests would start failing.

> The downside of a small number would be that visualizer would be flooded with this message.

In my limited experience visualizer is always overwhelming. A more robust approach would it be add filtering capabilities to make it easier to look at the events of interest, or nodes of interest. Default filters would solve the problem of too many of any event.

Reply:
Some state_sync tests were failing even with 200ms so I changed it back to 100ms.